executor/collation_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor
+
+import (
+	. "github.com/pingcap/check"
+	"github.com/pingcap/parser/mysql"
+	"github.com/pingcap/tidb/expression"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/chunk"
+	"github.com/pingcap/tidb/util/collate"
+	"github.com/pingcap/tidb/util/mock"
+)
+
+var _ = SerialSuites(&testCollationSuite{})
+
+type testCollationSuite struct {
+}
+
+func (s *testCollationSuite) TestVecGroupChecker(c *C) {
+	collate.SetNewCollationEnabledForTest(true)
+	defer collate.SetNewCollationEnabledForTest(false)
+
+	tp := &types.FieldType{Tp: mysql.TypeVarchar}
+	col0 := &expression.Column{
+		RetType: tp,
+		Index:   0,
+	}
+	ctx := mock.NewContext()
+	groupChecker := newVecGroupChecker(ctx, []expression.Expression{col0})
+
+	chk := chunk.New([]*types.FieldType{tp}, 6, 6)
+	chk.Reset()
+	chk.Column(0).AppendString("aaa")
```

Comment:
Please add some data for padding behaviors tests.

Reply:
Updated, PTAL.